SequenceOfNumbers.java

```diff
@@ -1,27 +1,89 @@
 import java.util.Scanner;
 
 public class SequenceOfNumbers {
-  public static void main(String[] args) {
-    System.out.println("Enter your sequence (use space between numbers):");
-    Scanner in = new Scanner(System.in);
-    String sequence = in.nextLine();
-    String numStr[] = sequence.split(" ");
-    
-    if (numStr.length == 1) {
-      System.out.println("You entered only one number. It is not a sequence");
-      return;
+
+  /**
+   * The function is used for determination type of sequence.
+   * It may be non-decreasing or not non-decreasing.
+   *
+   * @param newArgs - array of String type which contain testing sequence.
+   * One elemet of array - one number.
+   */
+  public boolean сheckSequenceIsNonDecreasing(String[] newArgs) {
+    for (int i = 0; i + 1 < newArgs.length; i++) {
+      if (Integer.parseInt(newArgs[i]) > Integer.parseInt(newArgs[i + 1])) {
+        return false;
+      }
+    }
+    return true;
+  }
+
+  /**
+   * This function is used for checking are the entered numbers a sequence.
+   *
+   * @param newArgs - array of String type which contain testing sequence.
+   * One elemet of array - one number.
+   *
+   * @return true if was entered several numbers,
+   * and false if entered only one number.
+   */
+  public boolean checkIsOneSymbol(String[] newArgs) {
+    if (newArgs.length == 1) {
+      return true;
+    }
+  return false;
+  }
+
+  /**
+   * This function is used for calling functions
+   * {@link SequenceOfnumbers#сheckSequenceIsNonDecreasing}
+   * and {@link SequenceOfnumbers#checkIsOneSymbol}.
+   *
+   * @param newArgs - array of String type which contain testing sequence.
+   * One elemet of array - one number.
+   * 
+   * @return string with information about sequence.
+   *
+   * @exception NumberFormatException is thrown
+   * when user enter wrong data.
+   */
+  public String generalCheck(String[] newArgs) {
+    final String POSITIVE_ANSWER = "Your sequence is non-decreasing";

```

Comment:
if it performs general check then it must know nothing about messages which should be output